mongoDb/dummyData.js

```diff
@@ -0,0 +1,97 @@
+const mongoDb = require('./index.js');
+const postgresDb = require('../postgresDb/index.js');
+
+// seed user movies table with users, recommended and currently watching movies
+
+// generate 20 movieIds
+const movies = [];
+for (let i = 0; i < 20; i += 1) {
+  // small chance of duplicate movies okay for dummy data
+  movies.push(Math.floor(Math.random() * 300000));
+}
+
+// retrieve movie objects by id for recs list
+function generateRecs(arr) {
```

Comment:
Maybe refactor using variables (const declarations) for better control over hoisting.

Reply:
done